### DISCUSSION TITLE
blog/cloud/2024-04-03-Container-and-Docker/

### DISCUSSION THREAD

**Discussion by giscus** [mingdodev/mingdodev.github.io#13] [Announcements]
# blog/cloud/2024-04-03-Container-and-Docker/

moonshot

https://mingdodev.github.io/blog/cloud/2024-04-03-Container-and-Docker/

**Comment by solsoleee** [comment]
잘 읽고 갑니다. 🎵
도커를 사용 하는 이유를 명확히 설명해주셨네요!
